### ISSUE TITLE
Port twisted.python.syslog to Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7957]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#7957|
|Type|enhancement|
|Created|2015-07-02 06:44:15Z|
|Branch|https://github.com/twisted/twisted/tree/7957-rodrigc-syslog|

Taking out of review queue for to rework

<details><summary>Searchable metadata</summary>

```
trac-id__7957 7957
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__Python_3_x Python-3.x
branch__7957_rodrigc_syslog 7957-rodrigc-syslog
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1435819455317516 1435819455317516
changetime__1467282125611993 1467282125611993
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/274

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_4"></a>|@Lukasa set owner to @rodrigc|
|-|-|

Alright, looks good to me. Transient and unrelated failure of the Python 3.4 builder. Merge when ready.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|GitHub set status to `closed`|
|-|-|

In changeset 81e7f0a43887afae9b9ea28e4322032d43edfe77
```
#!CommitTicketReference repository="" revision="81e7f0a43887afae9b9ea28e4322032d43edfe77"
Merge 7957-rodrigc-syslog: Add twisted/python/syslog.py to list of modules ported to Python 3

Author: rodrigc
Reviewer: Lukasa
Fixes: #7957
```
